operator/v1/types_clustermanager.go

```diff
@@ -119,6 +119,19 @@ type WorkConfiguration struct {
 	//  	he can set featuregate/Foo=false before upgrading. Let's say the cluster-admin wants featuregate/Foo=false.
 	// +optional
 	FeatureGates []FeatureGate `json:"featureGates,omitempty"`
+
+	// WorkDriver represents the type of work driver. Possible values are "kube", "mqtt", or "grpc".
+	// If not provided, the default value is "kube".
+	// If the driver is not "kube", the driver config must be provided in a secret named "work-driver-config" with the following structure:
```

Comment:
in the same ns as cluster manager is running, because this secret needs to be created by user, and `open-cluster-management-hub` ns is not created before clustermanager CR is created.